.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+dist: bionic
+python:
+  - "3.6"
+  - "3.7"
+script:
+  - python3 -m unittest discover
```

Comment:
We need to additionally check that the generated changes match the current repository state - that is, avoid Eiso moments in the future.

Reply:
yeah, anyway I want to check more then the current tests, but gonna update modelforge first